Cargo.toml

```diff
@@ -81,7 +81,8 @@ name = "value_operator"
 harness = false
 
 [features]
-default = []
+default = ["runtime-detection"]
```

Comment:
the runtime detection always has fewer overheads, I think it is better to not enable the feature in the default